samples/highcharts/demo/column-test/readme.md

```diff
@@ -0,0 +1,8 @@
+# Column test
+
+A [column chart](https://api.highcharts.com/highcharts/plotOptions.column), like the bar chart, is often used to
+visualize **comparisons** of data sets with discrete data, where the focus is on the values instead of categories as
+people read from left to right. This demo visualizes estimated corn vs wheat production for 2020.
```

Comment:
Probably best update `This demo visualizes estimated corn vs wheat production for 2020` suitably